documentation.md

```diff
@@ -176,7 +176,6 @@ Slides, videos and EC2-based exercises from each of these are available online:
 <h3>External Tutorials, Blog Posts, and Talks</h3>
 
 <ul>
-  <li><a href="http://engineering.ooyala.com/blog/using-parquet-and-scrooge-spark">Using Parquet and Scrooge with Spark</a> &mdash; Scala-friendly Parquet and Avro usage tutorial from Ooyala's Evan Chan</li>
```

Comment:
`http://engineering.ooyala.com/blog`  is not accessible, `Ooyala` should have been acquired by other companies, but no alternative link was found